yb-voyager/cmd/importData.go

```diff
@@ -1001,6 +1014,7 @@ func extractCopyStmtForTable(table string, fileToSearchIn string) {
 			utils.ErrExit("error while readline for extraction of copy stmt from file %q: %v", fileToSearchIn, err)
 		}
 		if copyCommandRegex.MatchString(line) {
+			line = strings.Trim(line, ";") + ` WITH (ROWS_PER_TRANSACTION %v)`
```

Comment:
Question: Do we have any cases where the `line` will already have a `WITH` clause in it? e.g. during `import data file`?

Reply:
yes thats the case with import data.
I have taken care of copyCommand generation in that case